STRINGS/REVERSE-A-STRING/C++/Reverse_String.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+#include <string.h>
+
+using namespace std;
+
+void reverse(char str[]){
+
+    int len = strlen(str);
+    char ch;
+
+    for(int i=0; i<(len/2); i++){
+
+        // swap ith character with 'length-i-1'th character
+        ch = str[i];
+        str[i] = str[len-i-1];
+        str[len-i-1] = ch;
+
+    }
+
+}
+
+int main(){
+
+    char str[20];
```

Comment:
use memory allocation using malloc for more memory optimization , hardcoded memory can cause failure of test cases.
like if a string exceeds 20th index.


Change:

include stdlib.h library
// take size as an input from the user for more flexibility
char * str = (char *) malloc ( size * sizeof(char));

Reply:
using **malloc** in Pallindrome.c
using **new** in Reverse_String.cpp _(since **new** is preferred in C++)_